src/commands/build.ts

```diff
@@ -14,6 +14,7 @@ interface BuildOptions {
   mode?: Mode
   output?: string
   deploy?: boolean
+  config?: object
```

Comment:
```suggestion
  config?: Record<string, string>
```

By using the [Record utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type), we can specify the value type in the `config` object.